File: app/src/main/java/com/zerobracket/sshcommadapp/network/ApiInterface.java
package com.zerobracket.sshcommadapp.network;

import com.zerobracket.sshcommadapp.models.SshLinkModel;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {
    @GET("ssh")
    Call<List<SshLinkModel>> getCommands();
}

File: app/src/main/java/com/zerobracket/sshcommadapp/network/ApiInterface.kt
package com.zerobracket.sshcommadapp.network

import com.zerobracket.sshcommadapp.models.SshLinkModel
import retrofit2.Call
import retrofit2.http.GET

interface ApiInterface {
    @get:GET("ssh")
    val commands: Call<List<SshLinkModel>>
}